<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadedItemWidget</class>
 <widget class="QWidget" name="DownloadedItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>146</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="outsideLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="titleLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="Spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
            <iconset theme=":/icon/delete.svg"/>
        </property>
        <property name="_vanillaStyle_Patch" stdset="0">
         <string>CleanButtonPatch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="btnWidget" native="true">
     <layout class="QHBoxLayout" name="btnLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelSize">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRemainingTime">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDuration">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRestart">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
            <iconset theme=":/icon/restart.svg"/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="_vanillaStyle_Patch" stdset="0">
         <string>CleanButtonPatch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFolder">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
            <iconset theme=":/icon/folder.svg"/>
        </property>
        <property name="_vanillaStyle_Patch" stdset="0">
         <string>CleanButtonPatch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDetail">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
            <iconset theme=":/icon/info.svg"/>
        </property>
        <property name="_vanillaStyle_Patch" stdset="0">
         <string>CleanButtonPatch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
